docs/advanced-configuration.md

```diff
@@ -108,6 +112,7 @@ isWindows: true
 image:
   otelcol:
     repository: quay.io/signalfx/splunk-otel-collector-windows
+    tag: <appVersion>-2022
```

Comment:
Opted not to document latest example since ideally tag bumps occur w/ upgrades only (and tracking latest can introduce breaking changes w/o notice).